web/source/dom/domManager.ts

```diff
@@ -1478,8 +1478,7 @@ namespace com.keyman.dom {
      * Description  KMW window initialization  
      */    
     init: (arg: com.keyman.OptionType) => Promise<any> = function(this: DOMManager, arg): Promise<any> { 
-      var i,j,c,e,p,eTextArea,eInput,opt,dTrailer,ds;
```

Comment:
Were all those variables just sitting there unused?!

Reply:
Yeah.  I've noticed them for a while, but always when doing totally unrelated work where it felt inappropriate to throw 'em in.